.github/workflows/add_release_label.yml

```diff
@@ -0,0 +1,62 @@
+name: Add release label to PR and linked issues when PR gets merged
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - closed
+
+jobs:
+  add_release_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0 # This is needed to checkout all branches
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install Yarn
+        run: npm install -g yarn
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Get the next semver version
+        id: get-next-semver-version
+        run: |
+          # Get the highest version from release branches
+          VERSION_BRANCHES=$(git branch -r | grep -o 'Version-v[0-9]*\.[0-9]*\.[0-9]*' | grep -o '[0-9]*\.[0-9]*\.[0-9]*' | sort --version-sort | tail -n 1)
+
+          # Get the highest version from tags
+          VERSION_TAGS=$(git tag | grep -o 'v[0-9]*\.[0-9]*\.[0-9]*' | grep -o '[0-9]*\.[0-9]*\.[0-9]*' | sort --version-sort | tail -n 1)
+
+          # Get the version from package.json
+          VERSION_PACKAGE=$(node -p "require('./package.json').version")
+
+          # Compare versions and keep the highest one
+          HIGHEST_VERSION=$(printf "%s\n%s\n%s" "$VERSION_BRANCHES" "$VERSION_TAGS" "$VERSION_PACKAGE" | sort --version-sort | tail -n 1)
+
+          # Increment the minor version of the highest version found
+          NEXT_VERSION=$(echo $HIGHEST_VERSION | awk -F. -v OFS=. '{$2++; print}')
+
+          # Check if FORCE_NEXT_SEMVER_VERSION is defined and not empty
+          if [ ! -z "${{ vars.FORCE_NEXT_SEMVER_VERSION }}" ]
```

Comment:
Nit: perhaps we could check this first and exit early if it's set, to skip all of the previous git operations if they aren't necessary.

e.g. at the top of this script:
```
if [ -n $FORCE_NEXT_SEMVER_VERSION ]
then
  echo "NEXT_SEMVER_VERSION=${FORCE_NEXT_SEMVER_VERSION}" >> $GITHUB_ENV
  exit 0
fi
```

Reply:
This is now fixed with this [commit](https://github.com/MetaMask/metamask-extension/pull/19061/commits/658bbef19085ffd87a4a6735ef0926ca61089314).